fern/docs.yml

```diff
@@ -74,206 +203,24 @@ navigation:
                 path: ./docs/pages/overview/define-your-api/ferndef/api-yml.mdx
               - page: Packages
                 path: ./docs/pages/overview/define-your-api/ferndef/packages.mdx
-              - page: Comparison with OpenAPI
-                path: ./docs/pages/overview/define-your-api/ferndef/comparison.mdx
-      - section: Fern CLI tool
-        slug: cli
-        contents:
-          - page: Overview
-            path: ./docs/pages/overview/cli/cli-overview.mdx
-          - section: CLI commands
-            slug: commands
-            contents:
-              - page: fern generate
-                path: ./docs/pages/overview/cli/cli-commands/fern-generate.mdx
-              - page: fern check
-                path: ./docs/pages/overview/cli/cli-commands/fern-check.mdx
-              - page: fern add
-                path: ./docs/pages/overview/cli/cli-commands/fern-add.mdx
-              - page: fern init
-                path: ./docs/pages/overview/cli/cli-commands/fern-init.mdx
-              - page: fern write-definition
-                path: ./docs/pages/overview/cli/cli-commands/fern-write-definition.mdx
-              - page: fern upgrade
-                path: ./docs/pages/overview/cli/cli-commands/fern-upgrade.mdx
-              - page: fern token
-                path: ./docs/pages/overview/cli/cli-commands/fern-token.mdx
-              - page: fern login
-                path: ./docs/pages/overview/cli/cli-commands/fern-login.mdx
-              - page: fern format
-                path: ./docs/pages/overview/cli/cli-commands/fern-format.mdx
-              - page: fern help
-                path: ./docs/pages/overview/cli/cli-commands/fern-help.mdx
-              - page: fern version
-                path: ./docs/pages/overview/cli/cli-commands/fern-version.mdx
-      - section: Changelog
+      - section: Fern CLI
```

Comment:
No overview page with install directions? This would be a natural place to look.